manifests/database.package.yaml

```diff
@@ -4,4 +4,4 @@ channels:
   name: stable
 - currentCSV: REPLACE_NAME.vREPLACE_NIGHTLY_VERSION-REPLACE_TAG
   name: nightly
-defaultChannel: stable
\ No newline at end of file
+defaultChannel: nightly
```

Comment:
@Avni-Sharma  why nightly ? we can still have the stable right ?

Reply:
I didnt want to disrupt the stable part, so thought of nightly . Shall I switch to stable?

Reply:
@pmacik suggestions?

Reply:
I've renamed the `nightly` channel to `beta` to match latest SBO, while keeping the `stable` for backwards compatibility with pre-v0.2.0 versions of SBO.